addons/web/static/src/js/views/list/list_editable_renderer.js

```diff
@@ -738,6 +739,34 @@ ListRenderer.include({
     _onFooterClick: function () {
         this.unselectRow();
     },
+    _onKeyDownAddRecord: function(e) {
+        switch(e.keyCode) {
+            case $.ui.keyCode.ENTER:
+                e.stopPropagation();
+                e.preventDefault();
+                this._onAddRecord(e);
+                break;
+        }
+    },
+    /** 
+     * It will returns the first visible widget that is editable
+     *
+     * @private
+     * @returns {Class} Widget returns first widget
```

Comment:
you can use the type Widget instead of class

Reply:
also, are we guaranteed to always have a return value? usually when we find something, null/undefined/-1 may happend...